bundle/edu.gemini.shared.skyobject/src/main/java/edu/gemini/shared/skyobject/Magnitude.java

```diff
@@ -124,7 +122,7 @@
      */
     public Magnitude(Band band, double brightness) {
         //noinspection unchecked
-        this(band, brightness, None.INSTANCE, System.DEFAULT);
+        this(band, brightness, None.INSTANCE, band != null ? band.defaultSystem : System.DEFAULT);
```

Comment:
If `band` is `null` the constructor will throw an `IllegalArgumentException`which just as well could be a `NullPointerException`.  In other words, I think just `band.defaultSystem` without the `null` check is okay.